builder/azure/arm/builder_acc_test.go

```diff
@@ -263,26 +320,28 @@ func TestBuilderAcc_rsaSHA2OnlyServer(t *testing.T) {
 	})
 }
 
-//Following functions are left in as they are part of newer Packer plugin acceptance testing framework
-//See https://github.com/hashicorp/packer-plugin-azure/pull/200#discussion_r879529490
-//nolint
-func testAuthPreCheck(t *testing.T) {
-	_, err := auth.NewAuthorizerFromEnvironment()
-	if err != nil {
-		t.Fatalf("failed to auth to azure: %s", err)
-	}
+type CreateSharedImageGalleryDefinitionParameters struct {
+	galleryImageName string
+	imageSku         string
+	imageOffer       string
+	imagePublisher   string
+	isX64            bool
+	isWindows        bool
+	useGenTwoVM      bool
+	specialized      bool
 }
 
-//nolint
-func checkTemporaryGroupDeleted(t *testing.T, b *Builder) {
+func createTestAzureClient(t *testing.T) AzureClient {
+	b := Builder{}
+	_, _, _ = b.Prepare()
 	ui := testUi()
-
+	// Use CLI auth for our test client
+	b.config.ClientConfig.UseAzureCLIAuth = true
```

Comment:
Hey Wilken, good catch on 1 I need to add 
something like
```	
if os.Getenv("AZURE_CLI_AUTH") == "" {
		t.Skip("Azure CLI Acceptance tests skipped unless env 'AZURE_CLI_AUTH' is set, and an active `az login` session has been established")
		return
	}
```

2.) We have other tests that test env variables, they are all defined in their respective packer templates, I just wanted to pick one type of auth for this setup/tear down rather than orchestrating something more complicated that checked for different types of provided auth, and CLI seemed the simplest for now

Reply:
Got it. I thought this was a requirement for all tests.